CHANGELOG.md

```diff
@@ -1,7 +1,10 @@
 ## Unreleased
 
-Features:
+Improvements:
```

Comment:
I think new chart options have been considered "features" in the past for this chart? Though looking at the changelog it doesn't seem very consistent in that respect 🤷.

Reply:
Yeah I think you're right though that features is the more common designation for the helm chart. I was thinking along the lines of the `vault` repo guidelines. I'll change this back as it's a separate discussion from this PR.